include/modem/lte_lc.h

```diff
@@ -1345,18 +1345,96 @@ int lte_lc_normal(void);
  * Requested periodic TAU (RPTAU) and requested active time (RAT) are used when PSM mode is
  * subsequently enabled using lte_lc_psm_req().
  *
- * For reference see 3GPP 27.007 Ch. 7.38.
+ * For encoding of the variables, see nRF AT Commands Reference Guide, 3GPP 27.007 Ch. 7.38., and
+ * 3GPP 24.008 Ch. 10.5.7.4a and Ch. 10.5.7.3.
  *
- * @param[in] rptau Requested periodic TAU as a null-terminated string.
- *                  Set to @c NULL to use manufacturer-specific default value.
- * @param[in] rat Requested active time as a null-terminated string.
- *                Set to @c NULL to use manufacturer-specific default value.
+ * @param[in] rptau
+ * @parblock
+ *          Requested periodic TAU as a null-terminated 8 character long bit field string.
+ *          Set to @c NULL to use manufacturer-specific default value.
```

Comment:
I had a better look at the AT command specification and the requested periodic TAU parameter is not optional after all. The spec does not say that omitting the value would use "manufacturer-specific" default, as it does say for the Active-Time parameter.

Reply:
As discussed offline, TAU is optional from the perspective that all parameters can be left out. It's true that AT reference guide doesn't give an option to give active time while not giving TAU. This works with the modem although AT guide doesn't mention it. If you think this requires further investigation, please create a ticket.